### ISSUE TITLE
Same level courses

### ISSUE THREAD

**Issue by ahmad007** [UniTime/unitime#157]
Using Unitime, what is the best practice to force the courses of the same level (e.g., Fall semester 1st-year courses) from being scheduled at the same time of the day, i.e., overlapped?

For instance, courses 101C, 102C, 103C, 104C, and 105C should be enrolled by the same group of students. Thus, these five courses should not be overlapped in the timetable. When I used the "Same time" distribution preference and prohibited these courses from being offered at the same time of the day, the solver couldn't find a solution, although I could do it manually easily. Of note, there are enough numbers of rooms and staff. 

In other words, I need to schedule the groups of courses at non-overlapped times. Fall semester 1st-year courses should not overlap with each other; similarly, Fall semester 2nd-year courses should not overlap with each other, etc.

**Comment by ahmad007** [comment]
Thanks for the feedback! But, where can I find the Required Different Time or Required Same Students distributions? I can't find these options in the distribution preferences.

**Comment by tomas-muller** [comment]
The distribution types are the Different Time (for classes that are not to
overlap in time) and the Same Students (for classes that cannot overlap in
time and cannot be too far from each other when placed back-to-back). The
Required is the preference level.

[image: Screenshot 2023-09-05 at 22.30.11.png]

Best regards,

Tomas Muller
UniTime
